scripts/gen-edpm-kustomize.sh

```diff
@@ -0,0 +1,152 @@
+    - op: replace
+      path: /spec/roles/edpm-compute/nodeTemplate/ansibleVars
+      value: |
+        edpm_sshd_allowed_ranges: ${EDPM_SSHD_ALLOWED_RANGES}
+        edpm_network_config_template: ${EDPM_NETWORK_CONFIG_TEMPLATE}
+        edpm_network_config_hide_sensitive_logs: false
+
+        neutron_physical_bridge_name: br-ex
+        neutron_public_interface_name: eth0
+        ctlplane_mtu: 1500
+        ctlplane_subnet_cidr: 24
+        ctlplane_gateway_ip: 192.168.122.1
+        ctlplane_host_routes:
+        - ip_netmask: 0.0.0.0/0
+          next_hop: 192.168.122.1
+        external_mtu: 1500
+        external_vlan_id: 44
+        external_cidr: '24'
+        external_host_routes: []
+        internal_api_mtu: 1500
+        internal_api_vlan_id: 20
+        internal_api_cidr: '24'
+        internal_api_host_routes: []
+        storage_mtu: 1500
+        storage_vlan_id: 21
+        storage_cidr: '24'
+        storage_host_routes: []
+        tenant_mtu: 1500
+        tenant_vlan_id: 22
+        tenant_cidr: '24'
+        tenant_host_routes: []
+        role_networks:
+        - InternalApi
+        - Storage
+        - Tenant
+        networks_lower:
+          External: external
+          InternalApi: internal_api
+          Storage: storage
+          Tenant: tenant
+
+        ctlplane_dns_nameservers:
+        - 192.168.122.1
+
+        dns_search_domains: []
+
+        edpm_ovn_metadata_agent_default_transport_url: ${EDPM_OVN_METADATA_AGENT_TRANSPORT_URL}
+        edpm_ovn_metadata_agent_metadata_agent_ovn_ovn_sb_connection: ${EDPM_OVN_METADATA_AGENT_SB_CONNECTION}
+        edpm_ovn_metadata_agent_metadata_agent_default_nova_metadata_host: ${EDPM_OVN_METADATA_AGENT_NOVA_METADATA_HOST}
+        edpm_ovn_metadata_agent_metadata_agent_default_metadata_proxy_shared_secret: ${EDPM_OVN_METADATA_AGENT_PROXY_SHARED_SECRET}
+        edpm_ovn_metadata_agent_default_bind_host: ${EDPM_OVN_METADATA_AGENT_BIND_HOST}
+        edpm_chrony_ntp_servers:
+        - ${EDPM_CHRONY_NTP_SERVER}
+        edpm_ovn_dbs:
+        - ${EDPM_OVN_DBS}
+
+        registry_name: quay.io
+        registry_namespace: tripleozedcentos9
+        image_tag: current-tripleo
+        edpm_ovn_controller_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-ovn-controller:{{ image_tag }}"
+        edpm_iscsid_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-iscsid:{{ image_tag }}"
+        edpm_logrotate_crond_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-cron:{{ image_tag }}"
+        edpm_nova_compute_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-compute:{{ image_tag }}"
+        edpm_nova_libvirt_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-libvirt:{{ image_tag }}"
+        edpm_ovn_metadata_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
+
+        extra_hosts_entries:
```

Comment:
it is applied to the Node CR but it doesn't seems to be applied to the edpm node. I don't know why.

Reply:
Maybe the difference is between the list of roles the [edpm-play.yaml](https://github.com/openstack-k8s-operators/install_yamls/pull/156/files#diff-c5f64cab8f76db38e0f0d1e3961e32ead7281c4328598f7dcca0fdf9f9051d8c) ran in the past compared to what roles the dataplane-operator runs now? Could you check this please

Reply:
weird, the vars are correctly set in the inventory

Reply:
edpm_play was using the `deploy-edpm-pre-network` playbook:
https://github.com/search?q=org%3Aopenstack-k8s-operators%20extra_hosts_entries&type=code
I believe if we rename the var to `edpm_hosts_entries_extra_hosts_entries` will fix the issue

Reply:
and I see we aren't importing the `edpm_hosts_entries` role in the dataplane-operator

Reply:
I renamed the vars and started this: https://github.com/openstack-k8s-operators/dataplane-operator/pull/115

Reply:
Awesome, now I was able to deploy a complete edpm node with `make edpm,_deploy`, boot a VM on it with nova with networking. \o/